README.md

```diff
@@ -1 +1,109 @@
-# webpack-setup-basics
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# MD Seafoods
+
+This website fetches seafoods meals gotten from an API [MealsDb](https://themealdb.com/browse.php?b=f) and display on the homepage.
+One can like a food by clicking on the heart icon and comment on a specific food by clicking on the comment button to have access to the comment form and place your comment in an instant.
+
+### ScreenShoot - Mobile screens
+
+![image](https://user-images.githubusercontent.com/50941074/147270738-aa0e9cb1-8fb3-4e16-9df1-614d636f1e05.png)
+
+### ScreenShoot - Desktop screens
+
+![image](https://user-images.githubusercontent.com/50941074/147270429-b2826c00-19b3-4fbb-bde1-a5d234117dd6.png)
+
+## Built With
+
+- HTML
+- CSS 
+- JavaScript ES6
+- Tailwind CSS 
+- Webpack-5
+- Jest
+
+## Live Demo
+
+[Live Demo](https://seafoods-app.netlify.app/)
+
+## Video Documentation
+[Video](https://drive.google.com/file/d/1yJMniajNASvKvGaUk6lF7yoetIP4ZV-C/view?usp=sharing)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Node and NPM
+- Web browser (Google chrome)
+- Code Editor (VSCode)
+
+### Development (Running locally)
+
+- git clone the project
+
+```bash 
+git clone git@github.com:AmaduKamara/seafoods-app
+```
+
+- Install Dependencies
+
+```bash
+npm install
+```
+
+To run StyleLint by itself, you may run the lint task:
+
```

Comment:
- [ ] Please provide the proper steps to run the project locally. After cloning there should be a command to enter into the folder and after installing the npm package, there should be a command to run the webpack server.

```
git clone git@github.com:AmaduKamara/seafoods-app

cd seafoods-app

npm i or npm install

// To run the server
npm start

```

Also, provide the command to test the project.

**`npm test`**

Reply:
Yeah sure. Thanks, @Dipeshtwis for picking that up. 🤝